playbooks/acs.yml

```diff
@@ -5,10 +5,11 @@
 - name: Handle secrets
   ansible.builtin.import_playbook: secrets.yml
 
-- hosts: all:!external
+- name: Gather facts from all hosts
+  hosts: all:!external
   gather_facts: no
   tasks:
-    - name: Gather facts from all hosts
+    - name: Setup
       ansible.builtin.setup:
```

Comment:
see https://github.com/Alfresco/alfresco-ansible-deployment/pull/260

it's because we want to gather all facts from all the hosts before everything else, and you can't have a run without at least a task, so to avoid gathering facts two times we ended up with this contruct